Tests/Realm.Tests/Database/TestObjects.cs

```diff
@@ -981,14 +981,15 @@ public partial class ObjectWithRequiredStringList : TestRealmObject
     public partial class ObjectWithEmbeddedProperties : TestRealmObject
     {
         [PrimaryKey]
-        public int PrimaryKey { get; set; }
+        [MapTo("_id")]
```

Comment:
I've slightly modified some classes for them to be compatible with sync, in order to not create new classes just for that.